build.proj

```diff
@@ -26,21 +26,27 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <DotnetCliBuildFrameworkInputs Include="build_projects/**/*.cs" />
-    <DotnetCliBuildFrameworkInputs Include="build_projects/**/*.json" />
+    <DotnetCliBuildFrameworkInputs Include="build_projects/**/*.cs" Exclude="build_projects/**/obj/**/*.cs" />
+    <DotnetCliBuildFrameworkInputs Include="build_projects/**/project.json;build_projects/**/project.lock.json" />
   </ItemGroup>
+  
+  <!-- Workaround to "Native image cannot be loaded multiple times" issue 
+       A target in the top level file needs to run and invoke a task -->
+  <Target Name="MSBuildWorkaroundTarget">
+    <Message Text="Dont remove this target" />
+  </Target>
```

Comment:
Can we get a URL to a GH issue here?

---
In reply to: [68872852](https://github.com/dotnet/cli/pull/3696#discussion_r68872852) [](ancestors = 68872852)